src/LanguageServer/Protocol/Protocol/SemanticTokens/SemanticTokensFullParams.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Roslyn.LanguageServer.Protocol
+{
+    using System;
+    using System.Text.Json.Serialization;
+
```

Comment:
We prefer usings outside namespaces.

```suggestion
using System;
using System.Text.Json.Serialization;

namespace Roslyn.LanguageServer.Protocol
{
```

Reply:
Please ignore me. Didn't realize we were in the Protocols library.